src/steps/iPressKey.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+	docs: {
+		description: "Fire a key press",
+		synopsis: "I press <key> [+<modifier>] at <control> [in <viewName> view]",
+		examples: ["I press ENTER at inputField", "I press 'F1' at inputField in Creation view", "I press ARROW_DOWN + ALT at myComboBox"]
+	},
+	icon: "edit",
+	regexp: new RegExp([
+    "^I press\\s",    //start of string
+    "(.+?)",          //key
```

Comment:
Use stricter RegExp - keys can be just sequences of upper case letters in the end...